src/types/interfaces.ts

```diff
@@ -336,6 +387,64 @@ export interface IMediaDeviceIds {
   outputDeviceId?: string | null;
 }
 
+/**
+ * Interface for updating the default media settings an SDK instance.
+ */
+export interface IMediaSettings {
```

Comment:
I actually started with this route, but like you said, the defaults were weird because we would only default it in the initial setup of the app. I could go either way here. Ultimately I ended up separating them so people wouldn't have to look up a separate definition when reading the sdk config.

Reply:
Sounds good. And yeah, I hate having to run down the rabbit trail of extended typings.